### DISCUSSION TITLE
Gradio CSS Thememaker?

### DISCUSSION THREAD

**Discussion by Woisek** [vladmandic/sdnext#260] [Q&A]
In the title. Is there a "Gradio CSS Thememaker" website, similar to what we have with jQueryUI (https://jqueryui.com/themeroller/)?
Or is there another way to visually create Gradio CSS styles?

**Comment by vladmandic** [comment]
check out <https://gradio.app/theming-guide/#using-the-theme-builder>

**Reply by Woisek** [reply]
Thanks!

>You can use the Theme Builder running on Spaces above, though **it runs much faster when you launch it locally via gr.themes.builder()**. 

Can we do this? 😳

**Reply by vladmandic** [reply]
I guess, I havent tried.
At the end, if theme is uploaded to gradio theme gallery, it should appear in the list of themes and you can apply it.

**Reply by Woisek** [reply]
Cool! So ... how do I start gr.themes.builder()? In the root folder via cmd.exe?

**Reply by vladmandic** [reply]
Naaah, that would be a new python project and you're set that in python source file.
If you're not comfortable with that, better use the online one.

**Reply by Woisek** [reply]
Oh, too bad. The online tool has a problem right now ...
Error

This space is experiencing an issue.

Please [contact the author of the space](https://huggingface.co/spaces/gradio/theme_builder/discussions/new?title=Space%20isn%27t%20working%20because%20there%20is%20a%20runtime%20error&description=Hello%2C%0A%0AFirstly%2C%20thanks%20for%20creating%20this%20space!%0A%0AI%20noticed%20that%20the%20space%20isn%27t%20working%20correctly%20because%20there%20is%20a%20runtime%20error.%0A%0AIt%20would%20be%20great%20if%20you%20could%20take%20a%20look%20at%20this%20because%20this%20space%20is%20being%20embedded%20on%20https%3A%2F%2Fgradio.app.%0A%0AThanks!) to let them know.

**Reply by vladmandic** [reply]
I guess it will be fixed soon...

**Reply by Woisek** [reply]
😥

**Reply by vladmandic** [reply]
I might add it to my repo as a side thing, but no promises.

**Reply by Woisek** [reply]
🎉🎉🎉

**Reply by Woisek** [reply]
Online Gradio Themer is still broken, since I week if I saw this correct ... 😔
Any chance to get this somewhere else onlilne?
